### ISSUE TITLE
[BUG] Each interest entry creates a new "symbol"

### ISSUE THREAD

**Issue by lildude** [ghostfolio/ghostfolio#3381]
**Important Notice**

The issue tracker is **ONLY** used for reporting bugs. New features should be discussed in our [Slack](https://join.slack.com/t/ghostfolio/shared_invite/zt-vsaan64h-F_I0fEo5M0P88lP9ibCxFg) community or in [Discussions](https://github.com/ghostfolio/ghostfolio/discussions).

Incomplete or non-reproducible issues may be closed, but we are here to help! If you encounter difficulties reproducing the bug or need assistance, please reach out to our community channels mentioned above.

Thank you for your understanding and cooperation!

**Bug Description**

Each interest entry for an account creates a new "symbol" which results in a very full and cluttered "Market data" page.

**To Reproduce**

1. Add two interest activities to an account
2. Browse to Admin control -> Market Data
3. Note there are two entries. Click each and note the different "symbols".

**Expected behavior**

Interest really shouldn't get a symbol but if it must because of design decisions, the same "symbol" should be used per account, as is the case for a normal stock.

Maybe interest shouldn't be on the market data page at all?

**Screenshots**

![Image](https://github.com/ghostfolio/ghostfolio/assets/627280/de345a06-fc28-48a8-a3e1-18dade4cd28b)

![Image](https://github.com/ghostfolio/ghostfolio/assets/627280/117167ab-a79f-474d-a0b0-7af396e975b2)

![Image](https://github.com/ghostfolio/ghostfolio/assets/627280/e59dd4f6-562d-4939-afae-1bb30a6cadb0)

**Logs**

N/A

**Environment**

- Ghostfolio Version 2.78
- Self-hosted
- Experimental Features enabled or disabled - none
- Firefox
- macos

**Additional context**

**Comment by dtslvr** [comment]
> Interest really shouldn't get a symbol but if it must because of design decisions, the same "symbol" should be used per account, as is the case for a normal stock.

Yes, each activity (_Order_) is associated to an asset profile (_SymbolProfile_), therefore reusing the asset profile per account could be cleaner. Would you like to work on that, @lildude?

**Comment by dickwolff** [comment]
@dtslvr I may have a repro on this, or at least an issue that is related. 

When importing multiple interests, randomly it generates a new symbol for interest.

I have this input (taken from the import file:
```json
{
    "accountId": "8996c276-de46-4fed-8f2d-9270681dbd49",
    "comment": "Interest on cash",
    "fee": 0.06,
    "quantity": 1,
    "type": "INTEREST",
    "unitPrice": 0.06,
    "currency": "EUR",
    "dataSource": "MANUAL",
    "date": "2024-05-11T21:02:44+02:00",
    "symbol": "Interest on cash"
},
{
    "accountId": "8996c276-de46-4fed-8f2d-9270681dbd49",
    "comment": "Interest on cash",
    "fee": 0.05,
    "quantity": 1,
    "type": "INTEREST",
    "unitPrice": 0.05,
    "currency": "EUR",
    "dataSource": "MANUAL",
    "date": "2024-05-12T21:02:49+02:00",
    "symbol": "Interest on cash"
},
```

When I import this multiple times, I get the following result. For some reason, the May 12th gets a correct name ("Interest on cash"), but the ones from May 11th gets a guid. Keeps happening to the May 11th, after importing multiple times.

![image](https://github.com/ghostfolio/ghostfolio/assets/5620002/7ef12aac-7d3f-4fa4-92fb-24f31dfa1232)

Any idea how this happens, and how to prevent it?

**Comment by juanabascal** [comment]
> each activity (Order) is associated to an asset profile (SymbolProfile), therefore reusing the asset profile per account could be cleaner.

Does this apply only for type interest or for other types as well? I see that INTEREST, FEE, ITEM and LIABILITY have the same logic when being created ([code pointer](https://github.com/ghostfolio/ghostfolio/blob/8319b216bb2faac7be0df305a286fadb2c022edb/apps/api/src/app/order/order.service.ts#L76-L93)).

**Comment by dtslvr** [comment]
> Does this apply only for type interest or for other types as well? I see that INTEREST, FEE, ITEM and LIABILITY have the same logic when being created.

Yes, not only for `INTEREST`. Also for other types as mentioned.

**Comment by csehatt741** [comment]
@dtslvr, I can take a look!
